src/os/posix/include/Task.h

```diff
@@ -0,0 +1,28 @@
+#ifndef __TASK_H__
+#define __TASK_H__
+
+#include <errno.h>
+#include <stdint.h>
+#include <time.h>
+
+namespace Task {
+    inline void delay(int ms) {
+        struct timespec ts;
+        int res;
+
+        ts.tv_sec = ms / 1000;
+        ts.tv_nsec = (ms % 1000) * 1000000;
+
+        do {
+            res = nanosleep(&ts, &ts);
+        } while (res && errno == EINTR);
```

Comment:
You can use sleepfor instead, which is compatible with the thread c++ api

https://en.cppreference.com/w/cpp/thread/sleep_for